ramalama/model.py

```diff
@@ -79,6 +78,21 @@ def remove(self, args):
     def symlink_path(self, args):
         raise NotImplementedError(f"symlink_path for {self.type} not implemented")
 
+    def gpu_args(self, model_path):
+        gpu_args = [ ]
+        if sys.platform == "darwin":
+            # llama.cpp will default to the Metal backend on macOS, so we don't need
+            # any additional arguments.
+            pass
+        elif sys.platform == "linux" and Path("/dev/dri").exists():
+            if "q4_0.gguf" not in model_path.lower():
```

Comment:
This is interesting... Should this condition apply to all GPU frameworks, CUDA, ROCm and Vulkan/Kompute or just Kompute?

Reply:
Only for kompute, so we need to make `model.py` aware of the variant it's running.

Reply:
I'd rather we just removed the q4_0.gguf check... If the model fails, it fails...

Many models are not named like this, none of the ollama ones are for example...

Reply:
I agree, lets remove the check.